### PR TITLE
feat: add support in fb pixel for advertiser_tracking_enabled

### PR DESCRIPTION
**Pull request by yashasvibajpai**
## What are the changes introduced in this PR?

We already support sending `advertiser_tracking_enabled` for facebook app events & facebook conversions. Here we are adding support for facebook pixel also.

Reference docs:
https://en-gb.facebook.com/business/help/365087497929660
https://developers.facebook.com/docs/marketing-api/conversions-api/app-events/
https://docs.tealium.com/server-side-connectors/facebook-conversions-connector/#:~:text=Advertiser%20Tracking%20Enabled

## What is the related Linear task?

Resolves INT-1419

## Please explain the objectives of your changes below
N/a

### Any changes to existing capabilities/behaviour, mention the reason & what are the changes ?

N/A

### Any new dependencies introduced with this change?

We are introducing a new property here. It would be set using context.device.adTrackingEnabled 

### Any new generic utility introduced or modified. Please explain the changes.

N/A

### Any technical or performance related pointers to consider with the change?

N/A

<hr>

### Developer checklist

- [ ] My code follows the style guidelines of this project

- [ ] **No breaking changes are being introduced.**

- [ ] All related docs linked with the PR?

- [ ] All changes manually tested?

- [ ] Any documentation changes needed with this change?

- [ ] Is the PR limited to 10 file changes?

- [ ] Is the PR limited to one linear task?

- [ ] Are relevant unit and component test-cases added?

### Reviewer checklist

- [ ] Is the type of change in the PR title appropriate as per the changes?

- [ ] Verified that there are no credentials or confidential data exposed with the changes.

### CODE REVIEW

**Comment by devops-github-rudderstack** [comment]
Test report for this run is available at: https://test-integrations-dev.s3.amazonaws.com/integrations-test-reports/rudder-transformer/3025/test-report.html

**Comment by devops-github-rudderstack** [comment]
This PR is considered to be stale. It has been open for 20 days with no further activity thus it is going to be closed in 7 days. To avoid such a case please consider removing the stale label manually or add a comment to the PR.

**Comment by devops-github-rudderstack** [comment]
This PR is considered to be stale. It has been open for 20 days with no further activity thus it is going to be closed in 7 days. To avoid such a case please consider removing the stale label manually or add a comment to the PR.

**Comment by devops-github-rudderstack** [comment]
This PR is considered to be stale. It has been open for 20 days with no further activity thus it is going to be closed in 7 days. To avoid such a case please consider removing the stale label manually or add a comment to the PR.

**Comment by devops-github-rudderstack** [comment]
This PR is considered to be stale. It has been open for 20 days with no further activity thus it is going to be closed in 7 days. To avoid such a case please consider removing the stale label manually or add a comment to the PR.

File: test/integrations/destinations/facebook_pixel/router/data.ts [hunk 1]
```diff
@@ -201,4 +201,113 @@ export const data = [
       },
     },
   },
+  {
+    name: 'facebook_pixel',
+    description: 'Test 1',
```
Comment (shrouti1507) [inline test/integrations/destinations/facebook_pixel/router/data.ts:206]: Please add proper description

**Review by shrouti1507 — APPROVED** [review]

**Review by ItsSudip — APPROVED** [review]

**Review by koladilip — APPROVED** [review]

File: test/integrations/destinations/facebook_pixel/router/data.ts (hunk 1 above)
Comment (yashasvibajpai) [reply test/integrations/destinations/facebook_pixel/router/data.ts]: Added, resolving this

**Pull request merged**